configs/frontend_config/config.json

```diff
@@ -1 +1,97 @@
-{}
\ No newline at end of file
+{
+  "@openmrs/esm-patient-chart-app": {
+    "extensionSlots": {
+      "patient-chart-dashboard-slot": {
+        "add": [
+          "nav-group#HIVCareTreatment"
+        ],
+        "configure": {
+          "nav-group#HIVCareTreatment": {
+            "title": "hIVCareAndTreatment",
+            "slotName": "hiv-care-and-treatment-group-slot",
+            "isExpanded": true
+          }
+        }
+      },
+      "hiv-care-and-treatment-group-slot": {
+        "add": [
+          "hiv-patient-summary-slot",
+          "program-management-slot",
+          "clinical-visits-slot",
+          "general-counselling-slot",
+          "partner-notification-slot"
+        ]
+      }
+    },
+    "Translation overrides": {
```

Comment:
We can't merge this in like this. This now overrides the translations for the whole patient chart app, so unless these are somehow name-spaced it will change translations of things other than what we are targeting.

Reply:
oh okay, I will take them back to the other configs.

Reply:
I was thinking that there may be a way to leverage Maven or something to automatically insert the block in the right place in the configs, but I don't really know how feasible that is.